### ISSUE TITLE
SoD P2 Books

### ISSUE THREAD

**Issue by Sommos** [Questie/Questie#5643]
# Druid
* Enhanced Restoration ❌
* Revive ❌
* Deeper Wilds ❌

# Hunter
* Aspect of the Viper ❌

# Mage
* Expanded Intellect ❌

# Paladin
* Enhanced Blessings ❌

# Priest
* Shadowfiend ❌
* Increased Fortitude ❌

# Rogue
* Redirect ❌

# Shaman
* Totemic Projection ❌

# Warlock
* Portal of Summoning ❌
* Soul Harvesting ❌

# Warrior
* Commanding Shout ❌

**Comment by BreakBB** [comment]
Hmmm... These are not related to any quest afaik, not sure if we should show the locations 🤔

**Comment by Sommos** [comment]
> Hmmm... These are not related to any quest afaik, not sure if we should show the locations 🤔

That is correct, and the majority seem to come from SM, however I think it may be useful for users to know where these are located as some act like runes such as the Priests 'Shadowfiend'.
